js/ui/components/clubhouse.js

```diff
@@ -567,9 +566,25 @@ var ClubhouseComponent = new Lang.Class({
                 return;
             }
 
-            this._oldAddNotificationFunc.apply(Main.notificationDaemon._gtkNotificationDaemon,
-                                               [params, invocation]);
+            oldAddNotificationFunc.apply(Main.notificationDaemon._gtkNotificationDaemon,
+                                         [params, invocation]);
         }
+
+        let oldRemoveNotificationFunc = GtkNotificationDaemon.prototype.RemoveNotificationAsync;
+        GtkNotificationDaemon.prototype.RemoveNotificationAsync = (params, invocation) => {
```

Comment:
Drive-by comment: This is a pre-existing problem and probably not an issue here because you have a singleton, but when you are defining methods directly on a prototype it's better to use `function` rather than arrow functions. That's because the arrow function binds the current value of `this` to the function. So if you were to call `otherDaemon.RemoveNotificationAsync()` it would not call the method on `otherDaemon`!